packages/visual-editor/scripts/insertI18n.ts

```diff
@@ -0,0 +1,141 @@
+import { Project, SyntaxKind } from "ts-morph";
+import prettier from "prettier";
+import fs from "fs";
+
+// CLI flag
+const isDryRun = process.argv.includes("--dry-run");
+
+/**
+ * Runs through everything under src/components and adds i18n("...") to all appropriate strings
+ */
```

Comment:
Does this wrap all strings or only those in YextField or label/title/description elements? Does it catch all React code like
```
<div>Some string</div>
```